Makefile

```diff
@@ -93,22 +91,8 @@ install-js-dev: node-version-check
 	# Add an additional check against `node_modules`
 	yarn check --verify-tree || yarn install --check-files
 
-install-py-dev: ensure-pinned-pip
-ifeq ($(shell command -v wheel 2> /dev/null),)
-	# The Python version installed via pyenv does not come with wheel pre-installed
-	# Installing wheel will speed up installation of Python dependencies
-	@$(PIP) install wheel
-endif
-	@echo "--> Installing Sentry (for development)"
-ifdef BIG_SUR
-	# grpcio 1.35.0 is very painful to compile on Big Sur, and is not really surfaced in testing anyways.
-	# So we set SYSTEM_VERSION_COMPAT=1 to get pip to download grpcio wheels for older MacOS.
-	SYSTEM_VERSION_COMPAT=1 $(PIP) install 'grpcio==1.35.0'
```

Comment:
Can we please keep this? I'd prefer if people didn't go and compile grpcio. Again, it takes forever, and is not applicable to testing.

Reply:
I take this into account in the if clause on the python.sh file by adding `SYSTEM_VERSION_COMPAT=1`:
```
    # Older versions of pip require SYSTEM_VERSION_COMPAT in Big Sur
    # shellcheck disable=SC2072
    if query_big_sur && [[ $pip_version < 20.3 ]]; then
        SENTRY_LIGHT_BUILD=1 SYSTEM_VERSION_COMPAT=1 pip install -e '.[dev]'
```

In the following output, I choose Python 3.8.7 and then try to install `grpcio` with an old pip version. I cancel it because it is downloading the source file to built it. Once I upgrade the pip version, it installs the wheel package w/o a problem. This is the same situation as with Python 3.6. `make install-py-dev` also works with 3.8 on Big Sur from scratch.
```
sentry on  armenzg/pip_SYSTEM_VERSION_COMPAT [$!?⇡] via ⬢ v12.19.0 via 🐍 v3.8.7
❯ python -m venv .venv

sentry on  armenzg/pip_SYSTEM_VERSION_COMPAT [$!?⇡] via ⬢ v12.19.0 via 🐍 v3.8.7 took 2s
❯ source .venv/bin/activate

sentry on  armenzg/pip_SYSTEM_VERSION_COMPAT [$!?⇡] via ⬢ v12.19.0 via 🐍 v3.8.7 (.venv)
❯ pip -V
pip 20.2.3 from /Users/armenzg/code/sentry/.venv/lib/python3.8/site-packages/pip (python 3.8)

sentry on  armenzg/pip_SYSTEM_VERSION_COMPAT [$!?⇡] via ⬢ v12.19.0 via 🐍 v3.8.7 (.venv)
❯ pip install grpcio==1.35.0
Collecting grpcio==1.35.0
  Downloading grpcio-1.35.0.tar.gz (21.2 MB)
     |████████████████████████████████| 21.2 MB 8.2 MB/s
Collecting six>=1.5.2
  Using cached six-1.15.0-py2.py3-none-any.whl (10 kB)
Using legacy 'setup.py install' for grpcio, since package 'wheel' is not installed.
Installing collected packages: six, grpcio
    Running setup.py install for grpcio ... /^canceled
ERROR: Operation cancelled by user
WARNING: You are using pip version 20.2.3; however, version 21.0.1 is available.
You should consider upgrading via the '/Users/armenzg/code/sentry/.venv/bin/python -m pip install --upgrade pip' command.

sentry on  armenzg/pip_SYSTEM_VERSION_COMPAT [$!?⇡] via ⬢ v12.19.0 via 🐍 v3.8.7 (.venv) took 8s
❯ pip install --upgrade pip
Collecting pip
  Using cached pip-21.0.1-py3-none-any.whl (1.5 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 20.2.3
    Uninstalling pip-20.2.3:
      Successfully uninstalled pip-20.2.3
Successfully installed pip-21.0.1

sentry on  armenzg/pip_SYSTEM_VERSION_COMPAT [$!?⇡] via ⬢ v12.19.0 via 🐍 v3.8.7 (.venv) took 2s
❯ pip install grpcio==1.35.0
Collecting grpcio==1.35.0
  Downloading grpcio-1.35.0-cp38-cp38-macosx_10_10_x86_64.whl (3.8 MB)
     |████████████████████████████████| 3.8 MB 2.1 MB/s
Requirement already satisfied: six>=1.5.2 in ./.venv/lib/python3.8/site-packages (from grpcio==1.35.0) (1.15.0)
Installing collected packages: grpcio
Successfully installed grpcio-1.35.0
sentry on  armenzg/pip_SYSTEM_VERSION_COMPAT [$!?⇡] via ⬢ v12.19.0 via 🐍 v3.8.7 (.venv)
❯ make install-py-dev
bash ./scripts/python.sh install-py-dev

You have explicitly set a non-recommended Python version (3.8). You're on your own.
...

```

Reply:
Oh, sorry. Didn't see. Thanks for clarifying!